corehq/ex-submodules/casexml/apps/phone/tasks.py

```diff
@@ -40,7 +40,7 @@ def force_update_cleanliness_flags():
 
 
 @task(serializer='pickle', queue=ASYNC_RESTORE_QUEUE)
-def get_async_restore_payload(restore_config):
+def get_async_restore_payload(restore_config, *args):
```

Comment:
Would be better to make these kwargs with comment so that readers know why they are here. I know you have a comment on the other side but someone looking at this function might not see that comment.

Reply:
updated. thanks